### ISSUE TITLE
Incompatible with optifine

### ISSUE THREAD

**Issue by qingchunnh** [embeddedt/ModernFix#41]
Incompatible with optifine
![optifine](https://user-images.githubusercontent.com/96961592/225090329-bd600214-f7c7-4a6d-ab58-acd4bb54a5f1.png)

**Comment by Kasualix** [comment]
Close-source mods are cursed.

**Comment by embeddedt** [comment]
There is important information missing here like the Minecraft version, but in any case, OptiFine is not supported. Some people have reported that ModernFix works with it on 1.16, but any issues that arise are unlikely to be fixed unless someone contributes a PR.

OptiFine increases launch times (often by several minutes in large packs) and is incompatible with many other content mods as well. I recommend using Rubidium as a more compatible and generally more performant alternative.

(It's also worth noting that this issue seems to simply be that the OptiFine localization strings are missing, I suspect most of its features will still work fine.)

**Comment by StormDragon-64** [comment]
It's features don't work (E.g ctm) too, it's not just the localization. I have this issue on 1.18.2 using Optifine U H9 pre2. I am [using my own modpack](https://www.curseforge.com/minecraft/modpacks/create-mod-plus) that I have recently added ModernFix to (in the most recent version, 4.10). 
Do you think that maybe a config change could fix this? I know you use sodium's config system but I couldn't find a list of options anywhere.

Here is a screenshot of the ctm not working and the issue shown in the above screenshot. You can see the glass is not connecting despite ctm being on. getting rid of modernfix fixes the issue.
![image](https://user-images.githubusercontent.com/80718549/225204082-58889ba9-0743-4c60-bc54-34204fe3a29b.png)

**Comment by embeddedt** [comment]
You would probably need to disable the resource pack caching option and that will nullify much of the benefit ModernFix provides (that option alone substantially shortens launch time). Because OptiFine is closed-source, I don't know what approach it uses to inject its CTM resources so supporting it will be hard.

**Comment by StormDragon-64** [comment]
There is, but it isn't compatible with rubidium

**Comment by StormDragon-64** [comment]
Turns out ctm itself still works, it's just that some of Optifine's default resources don't get loaded when using Modernfix.
So, I have created [this resource pack](https://drive.google.com/uc?export=download&id=1m9kzPmRXkpccpFuM9ObAzs5F-_HGiHQq) that makes Optifine work as usual without the need to disable the config above. This includes ctm, fancy grass, and the options translations.
Tested with Optifine 1.18.2 H9 pre 3 with many other client mods including ModernFix 2.2.2 and Lightspeed. Everything works as expected.
_Note: the pack was made for 1.18.2 so while it should work on other versions, you'll probably get a warning when installing it._
